sdk/eventhub/event-hubs/src/partitionPump.ts

```diff
@@ -158,3 +175,61 @@ export function getStartingPosition(
     return defaultEventPosition;
   }
 }
+
+/**
+ * @internal
+ * @ignore
+ */
+export function createProcessingSpan(
+  receivedEvents: ReceivedEventData[],
+  eventHubProperties: { eventHubName: string; endpoint: string },
+  tracingOptions: TracingOptions,
+  tracerLite: Pick<Tracer, "startSpan">
```

Comment:
why take the tracer as an argument instead of calling getTracer() inside this method?

Reply:
Makes it simpler to test - it's called _immediately_ before this though (I know we worry about it being stale).

Reply:
Gone anyways. It was so easy to use TestTracer now that it records all the little options.